src/title.ts

```diff
@@ -28,7 +23,27 @@ export interface TitleBase {
    */
   style?: string | string[];
 
-  // TODO: name, encode, interactive, zindex
+  /**
+   * A boolean flag indicating if the title element should respond to input events such as mouse hover.
+   */
+  interactive?: boolean;
```

Comment:
I don't think we expose `interactive` elsewhere in the codebase?

Reply:
Done